honeybot/plugins/translate.py

```diff
@@ -0,0 +1,340 @@
+# -*- coding: utf-8 -*-
+
+"""
+[weather.py]
```

Comment:
translate.py?

Reply:
Hi @Abdur-rahmaanJ , sorry about that i just made the changes now